src/lemlib/odom/differentialArc.cpp

```diff
@@ -96,10 +96,10 @@ float calcDeltaTheta(TrackingWheel& tracker1, TrackingWheel& tracker2) {
  * @note positive change in counterclockwise
  *
  * @param gyros vector of Gyro shared pointers
- * @return float the average change in heading
+ * @return Angle the average change in heading
  */
-float calcDeltaTheta(std::vector<std::shared_ptr<Gyro>>& gyros) {
-    float deltaTheta = 0;
+Angle calcDeltaTheta(std::vector<std::shared_ptr<Gyro>>& gyros) {
+    Angle deltaTheta = 0_deg;
```

Comment:
```suggestion
    Angle deltaTheta = 0_rad;
```
Change to radians for consistency